variants/devise/template.rb

```diff
@@ -79,6 +79,10 @@ def print_header(msg)
           /  # config.unlock_strategy = .+/,
           "  config.unlock_strategy = :email"
 
+gsub_file "config/initializers/devise.rb",
+          /  # config.parent_mailer = 'ActionMailer::Base',
+          "  config.unlock_strategy = 'ApplicationMailer'"
```

Comment:
config.parent_mailer

Reply:
@joshmcarthur I'm assuming there's a typo here?

Reply:
Given @joshmcarthur is on leave and this seems like a good fix to have in, I'll pick this up